cmd/system/aws_cli.go

```diff
@@ -0,0 +1,317 @@
+package system
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+	"time"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/Masterminds/semver"
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	execute "github.com/alexellis/go-execute/v2"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallAWSCLI() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "aws-cli",
+		Short: "Install AWS CLI",
+		Long:  `Install AWS CLI for interacting with Amazon Web Services APIs.`,
+		Example: `  arkade system install aws-cli
+  arkade system install aws-cli --version <version>`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringP("version", "v", "", "The version or leave blank to determine the latest available version")
```

Comment:
The default should be `githubLatest` like in the other system commands.
Check the [`firecracker`  command](https://github.com/alexellis/arkade/blob/master/cmd/system/firecracker.go#L56-L65) for example.